### ISSUE TITLE
请问数据增强之后，生成几张图呢？

### ISSUE THREAD

**Issue by loxoo6** [PaddlePaddle/PaddleSeg#3492]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

train_dataset:
  type: Dataset
  dataset_root: /home/aistudio/PaddleSeg/data
  train_path: /home/aistudio/PaddleSeg/data/train.txt
  num_classes: 3
  mode: train
  transforms:
    - type: ResizeStepScaling
      min_scale_factor: 0.5
      max_scale_factor: 2.0
      scale_step_size: 0.25
    - type: RandomPaddingCrop
      crop_size: [512, 512]
    - type: RandomHorizontalFlip
    - type: RandomDistort
      brightness_range: 0.5
      contrast_range: 0.5
      saturation_range: 0.5
    - type: Normalize

请问像这样的数据增强方法，输入一张图片后，会输出几张图呢？它是对每张图叠加使用这些数据增强方法，还是每一种数据增强方法都生成对应的图像呢？
paddleseg的数据增强可以把图片输出来嘛？

**Comment by Asthestarsfalll** [comment]
只生成一张经过叠加增强后的图像

**Comment by loxoo6** [comment]
> 只生成一张经过叠加增强后的图像

请问如何能多生成一些增强图呢，谢谢您

**Comment by Asthestarsfalll** [comment]
这应该是需要自己修改一下代码了，具体需要修改用到的transform，使其额外存储当前的增强图像
